.github/workflows/pr.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - run: npm i -g npm@6
```

Comment:
Temporarily running tests with specifically npm 6, until the other PR fixing the tests goes in